app/controllers/api/v1/notifications_controller.rb

```diff
@@ -1,8 +1,8 @@
 class Api::V1::NotificationsController < Api::V1::ApiController
   skip_before_action :authenticate_member!, only: [:create]
 
-  def show
-    @notification = current_member.notification
+  def index
+    @notification = Website.find(params[:website_id]).notification
```

Comment:
de ce nu folosesti `current_site`? face exact ce faci tu aici cu findu asta. In plus ce faci tu aici nu e sigur deloc.